extensions/wikia/ThemeDesigner/UploadBackgroundFromFile.class.php

```diff
@@ -35,26 +36,26 @@ public function verifyUpload(){
 	public function performUpload() {
 		global $wgUser;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)